frontend/.env.example

```diff
@@ -100,6 +100,9 @@ INTEROP_STATUS_CHECK_API_BASE_URI=
 # (optional; default: undefined)
 INTEROP_STATUS_CHECK_API_SUBSCRIPTION_KEY=
 
+# CDCP back end api
+CDCP_API_BASE_URI=https://api.example.com
```

Comment:
To follow conventions of other environment variables in this file.

```suggestion
# CDCP back end api
# (required; use the example below)
CDCP_API_BASE_URI=https://api.example.com
```

Reply:
done